amundsen_application/static/js/features/ColumnList/index.tsx

```diff
@@ -161,6 +163,7 @@ const ExpandedRowComponent: React.FC<ExpandedRowProps> = (
 
     return true;
   };
+  const uniqueValues = rowValue.stats && getUniqueValues(rowValue.stats);
```

Comment:
I would suggest something like:

```
const stats = rowValue.stats;
const normalStats = filterOutUniqueValues(stats);
const uniqueValueStats = getUniqueValues(stats);
```

Then we can remove the changes in the `ColumnStats` component and send `normalStats` instead. This moves the logic of splitting out the stats that need special processing and the normal stats all in one location that makes it easier to follow.

Reply:
That makes sense, thanks!